tests/test_entities.py

```diff
@@ -689,8 +639,8 @@ class CreatePayloadTestCase(TestCase):
     @classmethod
     def setUpClass(cls):
         """Set a server configuration at ``cls.cfg``."""
-        cls.cfg = config.ServerConfig('http://example.com')
-        cls.cfg_610 = config.ServerConfig('http://example.com', version='6.1.0')
+        cls.url = 'http://example.com'
+        cls.cfg = config.ServerConfig(cls.url)
```

Comment:
Since cls.url is not below, cant we keep that in one line ?
```suggestion
        cls.cfg = config.ServerConfig('http://example.com')
```